### DISCUSSION TITLE
Qt5 bottle failed to download

### DISCUSSION THREAD

**Discussion by gwerbin** [Homebrew/discussions#831] [Linux] [answered]
This is the output from `brew install --force-bottle --verbose qt`: <https://gist.github.com/gwerbin/e237a456b3677c8d655b8e3ff52dad36>

Is there supposed to be a bottle available for this version on Linux?

**Comment by SMillerDev** [comment, marked as answer]
It doesn't really have anything to do with the version of Linux. There should be a bottle available according to https://formulae.brew.sh/formula-linux/qt though.

@Homebrew/Linux ideas?

**Reply by iMichka** [reply]
The revision got lost in a merge (that's quite rare but there was probably a bigger merge conflict). I added it back (https://github.com/Homebrew/linuxbrew-core/commit/ee3e7ac8aecd066fc988c03b3d8026401ece67c8). You can `brew update` and try again.

**Reply by gwerbin** [reply]
It works, thank you!
